### ISSUE TITLE
Unable to cover view with image using resizeMode

### ISSUE THREAD

**Issue by reidkersey** [react-native-training/react-native-elements#1953]
I am trying to use resize mode to cover a view. With the standard RN Image, I can use the following code and it works fine. However using RNE, this same code fails. I assume this is due to the image being inside of a view in this component, however, even when I set the container to be the width and height (confirmed using background color) of the view I am tiring to fill, I am still unable to resize the image properly, and the image disappears entirely.

Working RN code:
`        <Image
          source={{ uri: logo }}
          resizeMode="cover"
          style={{
            width: undefined,
            height: undefined,
            flex: 1,
            backgroundColor: '#0F0',
          }}
        />`

Failing RNE Code:
`        <Image
          source={{ uri: logo }}
          resizeMode="cover"
          style={{
            width: "100%",
            height: undefined,
          }}
          containerStyle={{
            flex: 1,
            backgroundColor: '#0F0',
          }}
        />`

Is it possible to have an RNE Image component resize to fill a view?

**Comment by iRoachie** [comment]
You can’t do this. This is cause the image component from RNE has other views surrounding the image so using flex: 1 will make it fill only the parent view right above it.

**Comment by reidkersey** [comment]
That is what I figured. Is there a way to get an RNE image to 'cover' a view?

**Comment by iRoachie** [comment]
This is fixed in the upcoming release 1.2.0. in #1992. Thanks for bringing it up.

Will be able to use like this.

```jsx
<Image
  source={{
    uri: 'https://www.crockerriverside.org/sites/main/files/main-images/camera_lense_0.jpeg',
  }}
  style={{ ...StyleSheet.absoluteFillObject }}
  containerStyle={{ ...StyleSheet.absoluteFillObject }}
/>
```
